data/strings/strings.txt

```diff
@@ -32510,9 +32510,9 @@
     zh-Hant = iCloud 同步是一項正在開發中的實驗性功能。確保您已備份所有書籤和軌跡。
 
   [icloud_disabled_title]
-    comment = Title for the "iCloud is Disabled" alert.
+    comment = Title for the "iCloud Is Disabled" alert.
     tags = ios
-    en = iCloud is Disabled
+    en = iCloud Is Disabled
```

Comment:
capitalization

Reply:
You can read Apple's capitalization style guide here on page 42

https://help.apple.com/pdf/applestyleguide/en_US/apple-style-guide.pdf

they specifically say to capitalize "Is".

Reply:
I personally disagree but don't have a strong opinion and don't consider it to be wrong especially when the most important word is “Is"